Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ruby:3.0.1
+
+RUN apt-get update -q && apt-get install -qy vim
+
+RUN gem install jekyll
+
+WORKDIR /usr/src/app
+COPY . /usr/src/app
+RUN gem install webrick
```

Comment:
The versions of the dependencies need to be specific. The first version of the container worked fine with ruby 3.0.1 and jekyll. The latest version of jekyll apparently does not play well with ruby 3.0. I will be updating the Dockerfile with the specific versions to prevent jekyll from crashing inside the container.